templates/etcd-ha/3/docker-compose.yml

```diff
@@ -1,41 +1,22 @@
 etcd:
-  image: rancher/etcd:v2.3.6
+  image: rancher/etcd:v2.3.6-4
   labels:
     io.rancher.scheduler.affinity:container_label_ne: io.rancher.stack_service.name=$${stack_name}/$${service_name}
     io.rancher.sidekicks: data
-    io.rancher.service.allocate.skip.serialize: 'true'
-    io.rancher.container.start_once: 'true'
   environment:
     ETCD_DATA_DIR: /data
     ETCDCTL_ENDPOINT: http://etcd:2379
+    RANCHER_DEBUG: ${DEBUG}
   links:
   - data
-  - discovery
   volumes_from:
-    - data
-# containerize data volume to enable restarts and upgrades
+  - data
+
 data:
   image: busybox
-  command: /bin/true
+  entrypoint: /bin/true
```

Comment:
changing this will cause the data container to be destroyed on upgrade I believe. Is this needed?